apps/console.py

```diff
@@ -536,6 +546,15 @@ async def do_exit(self, params):
     async def do_quit(self, params):
         self.ui.exit()
 
+    async def do_filter(self, params):
+        if params[0] == "address":
+            if len(params) != 2:
+                self.show_error('invalid syntax', 'expected filter address <pattern>')
+                return
+            self.device.listener.address_filter = params[1]
+        if params[0] == "disable":
```

Comment:
since the `filter address <pattern>` takes a regexp, maybe `disable address` isn't necessary, you could just `filter address .*` ?

Reply:
removed disable address